.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
 tmp
+.DS_Store
```

Comment:
Ha, thanks for this. Those damn `DS_Store` files   __* shakes fist in the air *__